docs/workforce-management-framework/topics/pro-installing-core-dependencies.adoc

```diff
@@ -0,0 +1,20 @@
+[id='pro-installing-core-dependencies-{chapter}']
```

Comment:
This looks like overcomplicating things. 
IMHO we should  avoid splitting contents into separate files if not needed.
In my opinion we should do it only when content needs to be reused or it's from different type like concept etc. Doing this way will introduce a lot of additional work on our side. 
We will need to simplify this procedure to have just single command but then this will mean operating on multiple files etc. Let's avoid some heavy changes as often we are changing technical meaning.

From technical point of view after refactor entire procedure lost some information and it's little bit misleading. 

I will avoid headings totally. Just raw points - this procedure will be simplified later so we can just edit them in one place without jumping and checking if this is reused (it has little to no chance of being reused).